docs/UserGuide.adoc

```diff
@@ -202,19 +194,19 @@ image::demo/Open.png[]
 
 === Edit an existing note: `edit`
 
-If you want to change the `BODY` content of a note, use the `edit` command. This command *replaces* the existing `BODY` content of the currently open note, with the new `BODY` content. +
+If you want to change the `BODY` content of a note, use the `edit` command. This command *updates* the existing `BODY` content of the currently open note, with the new `BODY` content. +
 
-*Format:* `edit -b BODY`
+*Format:* `edit`
 
 [WARNING]
 ====
 * You are not allowed to `edit` the root note, i.e the currently opened path is ``"/"``
 ====
 
 ==== Example: Editing the currently opened note
-. Let's replace the `BODY` content of the currently opened note `Workspace`, with "This is the new paragraph". This can be done with the command: +
+. Let's update the `BODY` content of the currently opened note `Workspace`, with "This is the new paragraph". This can be done with the command: +
 
-    edit -b This is the new paragraph
+    edit
```

Comment:
OK, we should update this (since we're now using a modal), hahaha. I'll take note of this.

Reply:
Ok, will leave it to you!